nativelib/src/main/scala/java/lang/ieee754tostring/ryu/RyuRoundingMode.scala

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 Ulf Adams
```

Comment:
Please copy paste the whole license notice to LICENSE.md file (see previous entries over there for formatting).

Reply:
Done, as requested.  Tested locally by running sphinx-build and examining resultant html
file with Firefox.